pkg/gofr/datasource/cassandra/cassandra.go

```diff
@@ -296,8 +297,10 @@ type Health struct {
 	Details map[string]interface{} `json:"details,omitempty"`
 }
 
+var errStatusDown = errors.New("status down")
```

Comment:
Please move this error declaration upwards where other structs are being declared.